dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitorTest.kt

```diff
@@ -1097,6 +1097,20 @@ internal class DatadogRumMonitorTest {
         }
     }
 
+    @Test
+    fun `M delegate event to rootScope W sendWebViewEvent()`() {
```

Comment:
should we assert that session is updated after that?

Reply:
Oh...I think we are going to far, I think this is the `SessionScope` responsibility